doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -524,7 +524,7 @@ The Vernacular
    proof              : Proof . … Qed .
                       : | Proof . … Defined .
                       : | Proof . … Admitted .
-   decoration : #[ `attributes` ]
+   decoration : #[ `attribute` ]
    attributes : [`attribute`, … , `attribute`]
```

Comment:
The `attributes` non terminal has to go, unless we decide to keep both syntax `#[a] #[b]` and `#[a, b]` in which case it has to stay. I prefer the latter option.